src/devices/Ahtxx/samples/Ahtxx.Sample.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.Threading;
+
+namespace Iot.Device.Ahtxx.Samples
+{
+    /// <summary>
+    /// Samples for Ahtxx
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main entry point
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            const int I2cBus = 1;
+            I2cConnectionSettings i2cSettings = new I2cConnectionSettings(I2cBus, Aht20.DeviceAddress);
+            I2cDevice i2cDevice = I2cDevice.Create(i2cSettings);
+            Aht20 aht20Sensor = new Aht20(i2cDevice);
+
+            while (true)
+            {
+                Console.WriteLine($"{DateTime.Now.ToLongTimeString()}: {aht20Sensor.GetTemperature()}, {aht20Sensor.GetHumidity()}");
```

Comment:
It's better to be explicit on the unit when using UnitsNet. If you are just using `aht20Sensor.GetTemperature()` what will be printed is the unit it has been created with. Which is not that obvious. `aht20Sensor.GetTemperature().DegreesCelsius` is then much better as it's explicit and teach people how to properly use it. Someone who would prefer Kelvin or Fahrenheit would directly understand how to get them.
And same for humidity.

Reply:
@Ellerbach See my other comment above.